### PR TITLE
Update URLs for move to rpm-software-management GitHub organization

### PR DESCRIPTION
**Pull request by Conan-Kudo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,11 +40,11 @@ PARTICULAR PURPOSE. See the [GNU Lesser General Public License] for more details
 
 [crate-image]: https://img.shields.io/crates/v/librpm.svg
 [crate-link]: https://crates.io/crates/librpm
-[build-image]: https://travis-ci.com/RustRPM/librpm.rs.svg?branch=master
-[build-link]: https://travis-ci.com/RustRPM/librpm.rs/
+[build-image]: https://travis-ci.com/rpm-software-management/librpm.rs.svg?branch=master
+[build-link]: https://travis-ci.com/rpm-software-management/librpm.rs/
```
Comment (tarcieri) [inline README.md]: Will these work?

Comment (Conan-Kudo) [reply README.md]: I'm going to guess no, since it's not doing anything right now...

**Comment by Conan-Kudo** [comment]
@dmach I think somehow the Travis CI app isn't activated org-wide. Can you please fix this? 😦

**Comment by dmach** [comment]
@Conan-Kudo not sure if it's possible to activate travis ci org-wide.
I've activated travis for all projects one by one. I hope it helps.

**Comment by Conan-Kudo** [comment]
@dmach I believe there's a GitHub App request you need to allow for this to work as expected org wide. I made the request yesterday...

**Comment by Conan-Kudo** [comment]
@tarcieri I think we're all set now. I don't know why GitHub Checks for Travis CI doesn't work anymore, but at least Travis CI itself is working.

**Comment by tarcieri** [comment]
The checks probably have different status endpoints now that it’s travis-ci.org

**Comment by Conan-Kudo** [comment]
@tarcieri Probably. Sadly, it's not well-documented on Travis CI's website.

**Pull request merged**